src/ResourceManager/ContainerRegistry/Commands.ContainerRegistry/Commands/UpdateAzureContainerRegistry.cs

```diff
@@ -0,0 +1,75 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.ResourceManager.Common.Tags;
+
+namespace Microsoft.Azure.Commands.ContainerRegistry
+{
+    [Cmdlet(VerbsData.Update, ContainerRegistryNoun), OutputType(typeof(PSContainerRegistry))]
+    public class UpdateAzureContainerRegistry : ContainerRegistryCmdletBase
```

Comment:
@djyou this cmdlet should implement `SupportsShouldProcess` since it is making changes to an object in Azure